CHANGELOG.md

```diff
@@ -1,11 +1,26 @@
-10.4.1-rc.3 Release notes (2021-5-10)
+x.x.x Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
 * None.
 
 ### Fixed
 * Fixed the naming of `url` (now `baseUrl`) property on an app config to match the TypeScript declaration and other SDKs. ([#3612](https://github.com/realm/realm-js/issues/3612))
```

Comment:
Was it correct to move this from `10.4.1-rc.3` to `x.x.x`? Is this just fixing a bug that it was in the wrong place before, or are you introducing a bug by moving it?

Reply:
I forgot to add the template when I merged a PR yesterday so I am fixing it now.